docs/sources/dashboards/variables/variable-syntax/index.md

```diff
@@ -184,6 +184,16 @@ String to interpolate: '${servers:text}'
 Interpolation result: "test1 + test2"
 ```
 
+### Text to uppercase
+
+Formats single- and multi-valued variables into their uppercase text representation. For a single variable it will just return the uppercase text representation. For multi-valued variables it will return the uppercase text representation combined with `+`.
```

Comment:
I know this isn't like the other entries, but I'm updating the wording to follow our style guide. The page as a whole will eventually be edited for that as well.
```suggestion
Formats single- and multi-valued variables into their uppercase text representation. For a single variable it just returns the uppercase text representation. For multi-valued variables it returns the uppercase text representation combined with `+`.
```